### DISCUSSION TITLE
Repeat a request to an endpoint while applying a nuclei template

### DISCUSSION THREAD

**Discussion by Sim4n6** [projectdiscovery/nuclei-templates#7351] [Ideas]
Is there a way to **repeat a request to an endpoint while applying a nuclei template** to it, please?

**Comment by Sim4n6** [comment]
I mean an HTTP request to the endpoint `/path/abc` goes through **proxify** for instance and get **repeated**. 
But then I want to **fire the same request**  (with cookies, headers, body) but **manipulated** for instance to `/path/not-abc`.

What do you think please ? how should I proceed ?

**Comment by ehsandeep** [comment]
@Sim4n6 This is more of using proxify + nuclei in combination, something we have plans for but no immediate development; feel free to share more use cases for prioritization.

**Reply by Sim4n6** [reply]
Thank you for the clarification
